### ISSUE TITLE
Cannot send transaction with cardano-sl 1.3.2

### ISSUE THREAD

**Issue by v1nc3ntlaw** [input-output-hk/cardano-sl#3954]
My wallet already struggle with this issue for couple days. Even failed at Mac desktop wallet Daedalus with recent version package. So I try to restore my wallet at running up cardano-sl node. And get failed messages below.

```
[legacyServantBackend:DEBUG:ThreadId 419854] [2018-12-12 08:22:51.04 UTC] Transaction processing failed: 25006fb630d45cdc
[legacyServantBackend:INFO:ThreadId 419854] [2018-12-12 08:22:51.04 UTC] Transaction #25006fb630d45cdc application failed (SaveTxToilFailure (ToilTooLargeTx (Byte 32214) (Byte 8192)) - fatal)
[wallet.api:INFO:ThreadId 419852] [2018-12-12 08:22:51.05 UTC]
    Response #47 InternalError "Cannot send transaction: transaction's size exceeds limit (31.459 KiB > 8 KiB)" 0.589026703s
InternalError "Cannot send transaction: transaction's size exceeds limit (31.459 KiB > 8 KiB)"

[legacyServantBackend:DEBUG:ThreadId 420082] [2018-12-12 08:25:45.48 UTC] Transaction processing failed: 23f523118b95d67e
[legacyServantBackend:INFO:ThreadId 420082] [2018-12-12 08:25:45.48 UTC] Transaction #23f523118b95d67e application failed (SaveTxToilFailure (ToilTooLargeTx (Byte 32214) (Byte 8192)) - fatal)
[wallet.api:INFO:ThreadId 420079] [2018-12-12 08:25:45.49 UTC]
    Response #48 InternalError "Cannot send transaction: transaction's size exceeds limit (31.459 KiB > 8 KiB)" 0.65541419s
InternalError "Cannot send transaction: transaction's size exceeds limit (31.459 KiB > 8 KiB)"
```

**Comment by vantuz-subhuman** [comment]
Yep, this is a known problem atm ☹️ Tx size limit is 8 kb, so if tx contains "too much" inputs or outputs - it can be processed =(

**Comment by vantuz-subhuman** [comment]
basically it's something like this:
```
// signed tx
[
  //unsigned tx
  [
    // inputs
    [
      [0, tx-pointer],
      ...
    ],
    // outputs
    [
      [[address, checksum], value],
      ...
    ],
    // metadata
    {}
  ],
  // witnesses
  [
    [0, [pub-key, signature]],
    ...
  ]
]
```

**Comment by v1nc3ntlaw** [comment]
I even try to send very small amount 30 still failed. :(

**Comment by vantuz-subhuman** [comment]
I will try to look for any possible solution for this and write back. This is a really unpleasant issue, tho, I understand and agree =(

**Comment by v1nc3ntlaw** [comment]
My problem wallet still cannot make transaction on MacOS Daedalus client. But finally make transaction on setup and running up cardano-sl node through wallet transaction api.

In the first it was still failed due to I use groupingPolicy with OptimizeForSecurity value. Just think for the meaning was more secure so use it.

Since the problem related with transaction inputs structure. I saw the api doc says groupingPolicy was strategy to use for selecting the transaction inputs

So I try to change groupingPolicy use OptimizeForHighThroughput value. And now finally make successful send transaction from the wallet.

https://cardanodocs.com/technical/wallet/api/v1/#tag/Transactions%2Fpaths%2F~1api~1v1~1transactions%2Fpost

Hope write back here can help others if struggle with same problem.

**Comment by v1nc3ntlaw** [comment]
Updated here again. New released Daedalus Version 0.12.0, Release date: 18th of December 2018 fixed my issue too. I'm going close this issue.
